kickstarts/base.ks.erb

```diff
@@ -3,6 +3,7 @@
 logging --level=debug
 
 #version=DEVEL
+text
```

Comment:
Whoops, with *before* I understood immediately before this change. It may be that we did in the past!